### PR TITLE
update nodejs installation to version 7

### PR DESCRIPTION
**Pull request by tiagofilipe12**
Updated nodejs to version 7 instead of 6.

### CODE REVIEW

**Comment by tiagofilipe12** [comment]
Also, removed some useless parts of the code from bionode-watermill docker.

**Comment by tiagofilipe12** [comment]
Note that we should squash this commits into one when merging with master.

**Comment by tiagofilipe12** [comment]
Also added badges to the repos that have automated builds

File: bionode-ncbi/Dockerfile [hunk 1]
```diff
@@ -6,8 +6,13 @@ MAINTAINER Bruno Gonçalves, bfgoncalves@medicina.ulisboa.pt
 WORKDIR /NGStools/
 #INSTALL DEPENDENCIES
 RUN apt-get update
-RUN apt-get install -y curl sudo
+RUN apt-get install -y npm sudo curl
```
Comment (miguelpmachado) [inline bionode-ncbi/Dockerfile:9]: Do you really need `sudo`?

**Review by miguelpmachado — APPROVED** [review]

File: bionode-ncbi/Dockerfile (hunk 1 above)
Comment (tiagofilipe12) [reply bionode-ncbi/Dockerfile:9]: Yes!

**Pull request merged**